--- dmenu-5.3/config.def.h	2024-03-19 11:25:09.000000000 +0000
+++ dmenu-5.3.orig/config.def.h	2025-01-24 13:49:49.224554000 +0000
@@ -4,7 +4,7 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"FiraCode Nerd Font Mono Braille:style=Regular:size=12"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
